File: client/src/app/_dtos/hr/interviewAttendanceToUpdateDto.ts
export interface IInterviewAttendanceToUpdateDto
{
    interviewCandidateId: number;
    reportedAt: Date;
    interviewedAt: Date;
    interviewStatus: string;
    interviewStatusDate: Date;
    interviewRemarks: string;
}

export class InterviewAttendanceToUpdateDto implements IInterviewAttendanceToUpdateDto
{
    interviewCandidateId=0;
    reportedAt = new Date;
    interviewedAt = new Date;
    interviewStatus = '';
    interviewStatusDate = new Date;
    interviewRemarks = '';
}

File: api/DTOs/HR/InterviewAttendanceToUpdateDto.cs
namespace api.DTOs.HR
{
    public class InterviewAttendanceToUpdateDto
    {
        public int InterviewCandidateId { get; set; }
        public DateTime? ReportedAt { get; set; }  
        public DateTime? InterviewedAt { get; set;}
        public string InterviewStatus { get; set; }
        public DateTime InterviewStatusDate { get; set; }
        public string InterviewerRemarks { get; set; }
        public string AttachmentFileNameWithPath { get; set; }
    }
}